.github/workflows/property-polybft.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: PolyBFT Property tests
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - main
+      - develop  
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      E2E_TESTS: true
+      E2E_LOGS: true
+      E2E_TESTS_TYPE: 'Property Based'
+      CI_VERBOSE: true
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
Why do we need submodules?

Reply:
the same reason as for e2e

Reply:
Well, that's the thing: I don't find it necessary anymore. We don't need `core-contracts` to be cloned, because we are delivering SC in Edge codebase.

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1276/commits/cfee5ebbf083daee2151b1bd0f39b5e7fa7fcdff